### PR TITLE
Add TransferFunds API page

### PR DESCRIPTION
**Pull request by petitnau**
Work on #2.
immagine
immagine

### CODE REVIEW

File: docs/api/algorand/transfer-funds.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+### Validation errors
+
+Returned when the parameters are invalid
+
+|     Name    |                              Content                              |
+| ----------- | ----------------------------------------------------------------- |
+| Status code | 400 																															|
+| `errorId` 	| `INVALID_PARAMS` 																									|
+| `message` 	| `Validation errors in CreateDelegatedPayment request.` 						| 
+| `errors` 		| Array of strings containing additional informations 							|
```
Comment (nikitaeverywhere) [inline docs/api/algorand/transfer-funds.mdx]: * information. Try to also pay attention to ending sentences with periods and without (be consistent)

File: docs/api/algorand/transfer-funds.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+| `errorId`		| `RECEIVER_NOT_OPTED_IN` 																					|
+| `message`		| `Receiver must opt in to the asset before receiving funds.` 			|
+
+## Example Usage
+
+```js
+const response = await fetch('https://api.decash.com/TransferFunds', {
+  method: 'POST',
+  body: JSON.stringify({
+		'assetId': 'usdd',
```
Comment (nikitaeverywhere) [inline docs/api/algorand/transfer-funds.mdx]: Wondering about the case too

**Review by nikitaeverywhere — APPROVED** [review]

File: docs/api/algorand/transfer-funds.mdx (hunk 2 above)
Comment (petitnau) [reply docs/api/algorand/transfer-funds.mdx]: Same as previous

File: docs/api/algorand/transfer-funds.mdx (hunk 1 above)
Comment (petitnau) [reply docs/api/algorand/transfer-funds.mdx]: Fixing it in all pages

Comment (nikitaeverywhere) [reply docs/api/algorand/transfer-funds.mdx]: \* information

**Pull request merged**